src/library.h

```diff
@@ -67,9 +67,29 @@ struct library_source
   int (*fullrescan)(void);
 
   /*
+   * Returns TRUE if the source supports items with the given virtual path
```

Comment:
The intention was to get the source that is responsible to handle a given virtual path (find_source in library.c). But i dropped this function and used the same logic that is present in library_scan_media (iterating over all sources and returning after the first source reports it could handle the request).